page-objects/pages/HMWRKHomeworkPage.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
 
 
     studentEditsTextHomeworkAndSubmits: async function () {
```

Comment:
I do not see a student editing anything here. The second thing is, should userRole be written in the name?

Reply:
this method will be deleted (refactored). It was a short time solution from @Grzesiek80 , I think it is good.

Reply:
will be refactored